src/System.AppContext/src/System/AppContext.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using Windows.ApplicationModel;
+
+namespace System
+{
+    public static class AppContext
```

Comment:
I was not aware of that... I am not sure how it can move as mscorlib requires it for back-compat switches.